docs/source/user_guide/troubleshooting.rst

```diff
@@ -44,3 +44,13 @@ Thus, the following cases will fail:
 - partially
 
   - pre-compilation of ``noarch`` packages, with no impact on capability to use the package but removing it will let artifacts (pycache) on the filesystem
+
+
+Hangs during install in QEMU
+----------------------------
+When using Mamba/Micromamba inside a QEMU guest, installing packages may sometimes hang forever due to an `issue with QEMU and glibc <https://gitlab.com/qemu-project/qemu/-/issues/285>`_. As a workaround, set ``G_SLICE=always-malloc`` in the QEMU guest, eg.::
+
+  export G_SLICE=always-malloc
+  mamba install ...
+
+See `#1611 <https://github.com/mamba-org/mamba/issues/1611>`_ for discussion.
```

Comment:
This is a horribly long thread. I scanned it to look for a summarizing comment we could link instead, but couldn't find anything. (Closest seems to me like https://github.com/mamba-org/mamba/issues/1611#issuecomment-1112970540, but not particularly conclusive.)

Reply:
Yeah... I will highlight this comment top of the PR.